### ISSUE TITLE
Failed to run benchmark code: failed to import name 'GJKVariant' from 'hppfcl' (on commit 2546b89)

### ISSUE THREAD

**Issue by OxDuke** [lmontaut/collision-detection-benchmark#3]
After successfully building and installing the code (on commit 2546b89) on a freshly installed Ubuntu20.04, I tried to run the benchmark using the following command:

However, I got the following error:
![Screenshot from 2022-07-06 17-02-51](https://user-images.githubusercontent.com/19476105/177513312-7310ca72-9d35-45c1-81dc-24c0b73ff867.png)

It seems that `GJKVariant` enum type was not imported to the `hppfcl` namespace. So I tried to import everything from `hppfcl` using the following command in the Python console:
`from hppfcl import *`
Then I printed out all the things in the Python environment using the following command:
`locals()`
Here is the result:

![Screenshot from 2022-07-06 17-07-18](https://user-images.githubusercontent.com/19476105/177514067-bfa07595-d4ac-49e5-abc1-06287eaea91e.png)

It seems that even though GJKVariant was binded in https://github.com/lmontaut/hpp-fcl/blob/340fe6a16b547901c260008824dc65ad6e96ab73/python/gjk.cc#L93, it was not imported somehow.

I am not very familiar with boost Python, so I am asking for help here.

**Comment by OxDuke** [comment]
I never found the cause of error. But after I tried everything from the start, the problem mentioned above seems to be fixed. 

However, I ran into another problem while running the Python demo:
I was doing this on a new docker image from ros:noetic
`docker pull ros:noetic && docker run -it ros:noetic`
Inside the docker, I installed everything (anaconda, tqdm, qhull, pinocchio and etc.) by following the instructions. 
But, when I tried to run the script by typing the following command in the terminal:
`python exp/continuous_ellipsoids/ellipsoids_quick_benchmark.py`
The following error was reported:
![Screenshot from 2022-07-14 18-16-19](https://user-images.githubusercontent.com/19476105/178961123-9c8894ef-8004-4c32-a63b-774f4493b623.png)

It seems to me that all the types do match, and I don't know why boost.Python is reporting such an error.
